eslint-rules/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-uilib",
-  "version": "1.0.28",
+  "version": "1.1.0",
```

Shono pacakge which brings us a depdency is locked using `~`
That means it gets only patches.. 
So if we want this new version to be taken we need to update the patch version and not minor. 

```suggestion
  "version": "1.0.29",
```